Kickstarter-iOS/SharedViews/PledgeDisclaimerView.swift

```diff
@@ -55,6 +56,11 @@ final class PledgeDisclaimerView: UIView {
     _ = ([self.iconImageView], self.leftColumnStackView)
       |> ksr_addArrangedSubviewsToStackView()
   }
+  
+  private func setupConstraints() {
+    self.iconImageView.setContentCompressionResistancePriority(.required, for: .horizontal)
+    self.iconImageView.setContentHuggingPriority(.required, for: .horizontal)
```

Comment:
Also, I'd like to hear your thoughts on splitting this change into a separate PR to keep this one cleaner, as this update caused a significant number of screenshot updates.

Reply:
Oh nice! Thanks for figuring that out! It definitely looks like these tests could've been flaky before. Yeah, I think it would be better to split this change out into a separate pr if that's not too difficult!